server/command_hooks.go

```diff
@@ -36,6 +36,8 @@ const (
 		"- `/dialog introduction-text` - Open an Interactive Dialog with optional introduction text. Once submitted, user's action is posted back into a channel.\n" +
 		"- `/dialog error` - Open an Interactive Dialog which always returns an general error.\n" +
 		"- `/dialog error-no-elements` - Open an Interactive Dialog with no elements which always returns an general error.\n" +
+		"- `/dialog field-refresh` - **NEW:** Open an Interactive Dialog with field refresh functionality (requires Apps Form feature flag).\n" +
+		"- `/dialog multistep` - **NEW:** Open a multi-step Interactive Dialog demonstrating form refresh on submit (requires Apps Form feature flag).\n" +
```

Comment:
Interesting that the feature flag is needed despite your changes in https://github.com/mattermost/mattermost/pull/33076. In wonder if we can drop that requirement.